.travis/setup

```diff
@@ -5,10 +5,8 @@ if [ -z "$INSTALL4J_LICENSE" ]; then
  exit -1
 fi
 
-export WINDOWS_32_JRE="windows-x86-1.8.0_66.tar.gz"
-export WINDOWS_64_JRE="windows-amd64-1.8.0_66.tar.gz"
-export MAC_JRE="macosx-amd64-1.8.0_66.tar.gz"
 export ASSET_URL="http://github.com/triplea-game/assets/blob/master/install4j"
+mkdir -p ~/.install4j6/jres
```

Comment:
install4j is installed by default to "~/install4j6". Is this path wrong, with the leading 'dot' on install4j6?